demos/bazel/.bazelrc

```diff
@@ -0,0 +1,5 @@
+# Required because by default Bazel strips source code info from proto file descriptors
+# (see https://github.com/bazelbuild/bazel/issues/3971)
+# and com.salesforce.reactivegrpc.gen.ReactiveGrpcGenerator does not generate any
+# services without source code info present
+build --protocopt=--include_source_info
```

Comment:
There are no errors, it just doesn't generate code when the info is not present. At [`ReactiveGrpcGeneratorL60`](https://github.com/salesforce/reactive-grpc/blob/483029679fa5bde2579232891be0e43186fb25a0/common/reactive-grpc-gencommon/src/main/java/com/salesforce/reactivegrpc/gen/ReactiveGrpcGenerator.java#L60) `getLocationList` returns an empty list, as a result the proto is not considered any further.

Reply:
🤔 The use of `Location`s is indeed sub-optimal. In the beginning of reactive-grpc, I was just trying to get something to work.

I have a hunch the generator could be refactored to not use location information. Is that something you would be willing to try? If not, we can merge as is.

Reply:
The new commit should make sure that comments simply aren't included if source code info isn't present.
I've tried to keep the generator similar as much as possible but it could probably do with some refactoring :)